docs/source/models.rst

```diff
@@ -257,21 +257,21 @@ This functionality is a convenient way of testing models before deploying to a r
 You deploy the Python Function flavor locally using the CLI interface to the :py:mod:`mlflow.pyfunc` module.
 The local REST API server accepts the following data formats as inputs:
 
-  * JSON-serialized Pandas DataFrames in the `split` orientation. For example,
-    `data = pandas_df.to_json(orient='split')`. This format is specified using a `Content-Type`
-    request header value of `application/json; format=pandas-split`. Starting in MLflow 0.9.0,
-    this will be the default format if `Content-Type` is `application/json` (i.e, with no format
+  * JSON-serialized Pandas DataFrames in the ``split`` orientation. For example,
+    ``data = pandas_df.to_json(orient='split')``. This format is specified using a ``Content-Type``
+    request header value of ``application/json; format=pandas-split``. Starting in MLflow 0.9.0,
+    this will be the default format if ``Content-Type`` is ``application/json`` (i.e, with no format
     specification).
 
-  * JSON-serialized Pandas DataFrames in the `records` orientation. **THIS FORMAT IS NOT
-    RECOMMENDED. It is not guaranteed to preserve column ordering.** Currently, this format is
-    specified using a `Content-Type` request header value of  `application/json; format=pandas-records`
-    or `application/json`. However, starting in MLflow 0.9.0, `application/json` will refer to the
-    `split` format instead. For forwards compatibility, we recommend using the `split` format or
-    specifying the `application/json; format=pandas-records` content type.
+  * JSON-serialized Pandas DataFrames in the ``records`` orientation. *We do not recommend using
+    this format because it is not guaranteed to preserve column ordering.* Currently, this format is
+    specified using a ``Content-Type`` request header value of  ``application/json; format=pandas-records``
+    or ``application/json``. Starting in MLflow 0.9.0, ``application/json`` will refer to the
+    ``split`` format instead. For forwards compatibility, we recommend using the ``split`` format
```

Comment:
@dbczumar In general we recommend against talking about features that are scheduled for future releases. What is the purpose of announcing this before it's available.

Reply:
@stbof The purpose of announcing this change in advance is to allow users to migrate their client code before we implement the change. Some client code will break in `0.9.0` if users do not make certain modifications, so we need to inform them and give them the chance to modify their code beforehand.